book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] Kindly consider creating a many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental, They should be implemented similar to the following:
```ruby
 def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```